src/queries/GetSystemProfileTeamQuery.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from '@apollo/client';
+
+export default gql`
+  query GetSystemProfileTeam($systemId: String!) {
```

Comment:
Would it make sense to include all the data needed for both roles and CedarSystem by using this query method - `detailedCedarSystemInfo`?

`type DetailedCedarSystem {
cedarSystem: CedarSystem!
roles: [CedarRole!]!
deployments: [CedarDeployment!]!
}`

This would alleviate the need to refetch/reload the data each time the team tab is selected and consolidate BE calls.

Reply:
@patrickseguraoddball I was just preferring more granular requests for the subpages, and not loading data until it's actually used from a user visit and ready for display from the backend. The mvp just needs the team data. I don't think deployments had every data point necessary last time I checked. The detail query is also slower.

The detailed system type also wasn't used anywhere yet, and the idea of it may be old. Checked with @ClayBenson94

Reply:
Is there a plan to use a query like detailed for all subpages? I assumed a general subpage load scheme. There could be some sort of retention so that it wouldn't have to load every time.